CommerceCloud/Integration/03_SalesforceParticularities.md

```diff
@@ -0,0 +1,170 @@
+[!!Manage the Salesforce connection](./02_ManageSalesforceConnection.md)
+[!!Manage the ETL mappings in DataHub](../../DataHub/Operation/01_ManageETLMappings.md)
+[!!User interface Connections](../UserInterface/01_Connections.md)
+
+# Salesforce product data particularities
+
+There are several particularities regarding the Salesforce product data that must be taking into account when establishing and using the Salesforce connection. These particularities are described in detail below.  
+
+## Master catalog
+
+A catalog is a product tree. You can assign certain nodes from a certain catalog to a product and thus divide it into different categories. The product data are part of the catalog. Every product in Salesforce must belong to a catalog. This catalog, who owns the product, is the master catalog. The product belongs to the master catalog but must not be assign to a category within the master catalog. The master catalog in Salesforce determines which fields are available for a product.   
+Regardless of which catalog a product belongs to, the product can be assigned to other catalogs as well.
+
+The master catalog who owns the products must be mapped in the *DataHub* module to the *Master Catalog (CommerceCloud Actind)* destination attribute.
+The ETL mapping determines which catalog is used as master catalog in Salesforce.   
+It is mandatory that the master catalog is set in all ETL mappings to the Salesforce attribute set, see [Configure the master catalog](./02_ManageSalesforceConnection.md#configure-the-master-catalog).
+
+In Actindo, all fields from Salesforce are available, as there exists only one single attribute set for Salesforce.    
+
+
+## Sites
+
+Salesforce offers the possibility to create different sites for the order import. In Actindo, these sites are created as sub-shops. That means that every single shop is a sub-account, available in the stock withdrawal matrix in the *Warehouse* module.
```

Comment:
Either we write SalesForce CommerceCloud or CommerceCloud
SalesForce is the owning company it's like writing Microsoft when describing windows or an office program

I would just write:
SalesForce CommerceCloud offers the possibility to handle several sites within one instance

Which means that every single **site** is a